src/components/QuoteText.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const QuoteText = (props) => (
+  <div>
+    <p>{props.quote.text}</p>
+    <h3>{props.quote.speaker}</h3>
+    <h4>{props.quote.movie}</h4>
```

Comment:
This is totally fine, but there's a lot of redundant code here. 3 copies of `props.quote`. Can you instead destructure text, speaker, and movie from props and quote so we use less code?